straxen/analyses/bokeh_waveform_plot.py

```diff
@@ -718,7 +718,7 @@ def _make_event_title(event, run_id, width=1600):
                              sizing_mode='scale_both',
                              width=width,
                              default_size=width,
-                             orientation='vertical',
+#                              orientation='vertical',
```

Comment:
@WenzDaniel is it okay if we remove this line or is there some other way we can get this working with the more resent versions of bokeh?

Reply:
Yes I think we can. So not vital enough from keeping things being updated.